client/src/pages/About/Overview/Stats.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import './Stats.scss';
+
+interface StatsData {
+    drug_claims: number;
+    drugs: number;
+    gene_claims: number;
+    genes: number;
+    interaction_claims: number;
+    interactions: number;
+    gene_categorization_claims: number;
+    gene_categorizations: number;
+    sources: number;
+    publications: number;
+}
+
+export const AboutStats: React.FC = () => {
+    const [stats, setStats] = useState<StatsData | null>(null);
+    const [loading, setLoading] = useState<boolean>(true);
+    const [error, setError] = useState<string | null>(null);
+
+    useEffect(() => {
+        // Fetch stats from counts controller endpoint
+        const fetchStats = async () => {
+            try {
+                const response = await fetch('http://localhost:3000/api/counts');
+                
+                if (!response.ok) {
+                    throw new Error('Failed to fetch data');
+                }
+
+                const data = await response.json();  
+                setStats(data);  
+                setLoading(false);  
+            } catch (error) {
+                setError('Failed to fetch data.');
+                setLoading(false);
+            }
+        };
+
+        fetchStats();
+    }, []);  
+
+    if (loading) {
+        return <div>Loading...</div>;
+    }
+
+    if (error) {
+        return <div>{error}</div>;
+    }
```

Comment:
@acoffman  @susannasiebert This block is intended to grab some live counts on data in DGIdb and display it for the user on this page. 

What should the `fetch` be set to for the eventual live instance? Currently the counts controller (introduced in https://github.com/dgidb/dgidb-v5/pull/598) renders this JSON on port 3000 (as opposed to the graphql controller on 3001). I'm concerned that just setting this to https://dgidb.org/api/counts won't work the way I think it will and this will break immediately upon publishing to main.

Thoughts?

Reply:
The `3000`/`3001` distinction goes away in production. 3000 is where the rails development server serves the backend and 3001 is where the react development server serves the react app. 

In production there is no "frontend" server, the react app is built into a static javascript file and served as a static asset, so there's only one port. (443 specifically, standard https).

Setting it to `https://dgidb.org/api/counts` will work fine for prod, though I think in general we've configured these via the .env files so they can vary between dev/prod/test etc rather than hardcoding in the component itself.